server/src/main/java/io/druid/metadata/SQLMetadataRuleManager.java

```diff
@@ -168,21 +170,26 @@ public void start()
         return;
       }
 
-      this.exec = Execs.scheduledSingleThreaded("DatabaseRuleManager-Exec--%d");
+      exec = MoreExecutors.listeningDecorator(Execs.scheduledSingleThreaded("DatabaseRuleManager-Exec--%d"));
 
       createDefaultRule(dbi, getRulesTable(), config.get().getDefaultRule(), jsonMapper);
-      ScheduledExecutors.scheduleWithFixedDelay(
-          exec,
-          new Duration(0),
-          config.get().getPollDuration().toStandardDuration(),
+      future = exec.scheduleWithFixedDelay(
           new Runnable()
           {
             @Override
             public void run()
             {
-              poll();
+              try {
+                poll();
+              }
+              catch (Exception e) {
+                log.error(e, "uncaught exception in rule manager polling thread");
```

Comment:
I don't think there should be an alert here without a retry strategy. An acute network hiccup should not cause an alert if it is simply recoverable.